include/Page/Base.php

```diff
@@ -222,7 +222,10 @@ public function modify_admin_footer_text( $footer_text ) {
 	 * Enqueue JavaScript.
 	 */
 	public function enqueue_script() {
-		wp_enqueue_script( 'bulk-move', $this->get_plugin_dir_url() . 'assets/js/bulk-move.js', array( 'jquery', 'postbox' ), BM_BulkMove::VERSION, true );
```

Comment:
Let's prevent formatting changes as part of other commits. They just add noise to diffs.